packages/roosterjs-editor-plugins/lib/plugins/TableSelection/TableSelection.ts

```diff
@@ -0,0 +1,195 @@
+import { EditorPlugin, IEditor, Keys, PluginEvent, PluginEventType } from 'roosterjs-editor-types';
+import {
+    Browser,
+    clearSelectedTableCells,
+    highlightTableSelection,
+    setTableSelectedRange,
+} from 'roosterjs-editor-dom';
+
+/**
+ * TableSelectionPlugin help highlight table cells
+ */
+export default class TableSelectionPlugin implements EditorPlugin {
+    private editor: IEditor;
+    private mouseUpEventListerAdded: boolean;
+    private lastTarget: EventTarget;
+    private firstTarget: EventTarget;
+
+    /**
+     * Get a friendly name of  this plugin
+     */
+    getName() {
+        return 'TableSelection';
+    }
+
+    /**
+     * Initialize this plugin. This should only be called from Editor
+     * @param editor Editor instance
+     */
+    initialize(editor: IEditor) {
+        this.editor = editor;
+        console.log(this.editor.getScrollContainer());
+        setTimeout(() => {
+            const styleElement = document.createElement('style');
+            styleElement.id = 'Style4TableSelection';
+            this.editor.getDocument().head.append(styleElement);
+
+            let sheet = styleElement.sheet;
+            // sheet.insertRule(
+            //     'td.TableCellSelected *::selection { background-color: red !important;}'
+            // );
+            // sheet.insertRule(
+            //     'td.TableCellSelected::selection { background-color: red !important;}'
+            // );
+
+            sheet.insertRule(
+                'td:not(td.TableCellSelected) *::selection { background-color: transparent !important;}'
+            );
+
+            sheet.insertRule(
+                'td:not(td.TableCellSelected)::selection { background-color: transparent !important;}'
+            );
+        }, 500);
+    }
+
+    /**
+     * Dispose this plugin
+     */
+    dispose() {
+        this.removeMouseUpEventListener();
+        this.editor = null;
+        document
```

Comment:
This will be an issue when:
1. projection popout
2. multiple instances of editor

Reply:
Suggest editor only manage things inside editor, but do not touch global window/document if we can